cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/DefaultStore.java

```diff
@@ -87,8 +92,12 @@
 public class DefaultStore implements Store {
   private static final Logger LOG = LoggerFactory.getLogger(DefaultStore.class);
   public static final String APP_META_TABLE = "app.meta";
```

Comment:
not necessarily your PR, but please group all the private static final fields together followed by the public static final fields